content/en/docs/concepts/containers/runtime-class.md

```diff
@@ -37,10 +37,10 @@ but with different settings.
 
 Ensure the RuntimeClass feature gate is enabled (it is by default). See [Feature
 Gates](/docs/reference/command-line-tools-reference/feature-gates/) for an explanation of enabling
-feature gates. The `RuntimeClass` feature gate must be enabled on apiservers _and_ kubelets.
+feature gates. The `RuntimeClass` feature gate must be enabled on the apiserver and the kubelet.
```

Comment:
I believe the existing text communicates this may need to be done on multiple nodes.  I don't think this change needs to be made.

Reply:
Thanks.

Reply:
It would be good to change 'apiserver' to 'API server'.

Reply:
Thank